### PR TITLE
Fix/add bts address generator

### PR DESCRIPTION
**Pull request by webersni**
- Changed bitcoin address type to segwit
- Added small script to get address pairs

### CODE REVIEW

File: pkg/address/address.ts [hunk 1]
```diff
@@ -1,9 +1,12 @@
 import * as bitcoin from "bitcoinjs-lib"
+import { PrivateKey, key } from "bitsharesjs"
+import { ChainConfig } from "bitsharesjs-ws"
+import crypto from "crypto-random-string"
 
 /**
  * An interface for a privatekey/address pair
  */
-export interface BTCAddress {
+export interface Address {
```
Comment (chronark) [inline pkg/address/address.ts:9]: Should we really rename this to a generic name?
Do we actually use Address for bitshares?

File: pkg/address/getAddress.test.ts [hunk 2]
```diff
@@ -0,0 +1,56 @@
+import { getAddress } from "./getAddress"
+import { mocked } from "ts-jest/utils"
+import { Address, getBTCAddress, getBTSAddress } from "./address"
+jest.mock("./address")
+
+describe("getAddress script", () => {
+    beforeEach(() => {
+        jest.resetAllMocks()
+        jest.restoreAllMocks()
```
Comment (chronark) [inline pkg/address/getAddress.test.ts:9]: As far as I know you don't need to restore after resetting, but I might be wrong.

Comment (webersni) [reply pkg/address/getAddress.test.ts]: Indeed, fixed it.

File: pkg/address/address.ts (hunk 1 above)
Comment (webersni) [reply pkg/address/address.ts:9]: Well, currently not. The address generator is for testing purposes only. However, it is possible to add new key pairs to a given Bitshares account and you should do that regularly.

**Pull request merged**